src/app/login/login.component.ts

```diff
@@ -64,7 +64,7 @@ export class LoginComponent implements OnInit {
           this.router.navigate([this.returnUrl]);
         },
         (error) => {
-          this.alertService.error(error.error.message);
+          this.alertService.error('Failed to login.');
```

Comment:
Ik denk dat we nog wel iets met de error message kunnen doen, er kunnen verschillende redenen zijn waarom het inloggen niet lukt (bijv. account bestaat niet, verkeerde wachtwoord, server offline). We zouden de message erachter kunnen plakken, of misschien zelfs op HTTP code checken

Reply:
Ik krijg de error 'No elements in sequence' wat komt uit .pipe(first()). Heb de error message iets veelzeggender gemaakt.